### PR TITLE
Fix sonar scanner

### PR DESCRIPTION
**Pull request by hanzala1234**

### CODE REVIEW

**Review by OsamaKhan220 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -85,9 +85,11 @@ RUN for f in "/etc/passwd" "/projects"; do \
 # Install sonarqube scanner
 RUN wget https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.2.0.1873-linux.zip && \
   unzip sonar-scanner-cli-4.2.0.1873-linux.zip  && \
-  mv sonar-scanner-4.2.0.1873-linux/bin/* /usr/local/bin
+
+  mv sonar-scanner-4.2.0.1873-linux /var/opt
+
 # install unzip & buildah
-RUN yum install unzip buildah -y
```
Comment (bilalbokharee) [inline Dockerfile:90]: @hanzala1234 why did you remove unzip?

Comment (hanzala1234) [reply Dockerfile:90]: if buildah could be installed with microdnf I would say to use that one
